pkg/service/service_test.go

```diff
@@ -140,3 +144,416 @@ func TestDeleteKubernetesComponentFromDevfile(t *testing.T) {
 		})
 	}
 }
+
+func TestListDevfileServices(t *testing.T) {
+	tests := []struct {
+		name       string
+		devfileObj parser.DevfileObj
+		wantKeys   []string
+		wantErr    error
+	}{
+		{
+			name: "No service in devfile",
+			devfileObj: parser.DevfileObj{
```

Comment:
How about we create a helper function to return the devfile data? I see it is repeated across the other tests.

Reply:
@feloy Are you going to work on this or do you think it would be better to keep them as is?

Reply:
I have made the change in my latest commit, see `getDevfileData`